python_modules/dagster/dagster/_core/definitions/decorators/assets_definition_factory.py

```diff
@@ -163,6 +164,28 @@ def build_subsettable_asset_ins(
     }
 
 
+def get_partition_mappings_from_deps(
```

Comment:
Why duplicate this here instead of importing the one in assets.py

Reply:
[![goodcatch](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/default-memes/goodcatch.png)](https://app.graphite.dev//settings/meme-library?org=dagster-io)